tools/e2etesting/TestEventProcessor/TestEventProcessor.Businesslogic/TestEventProcessor.Businesslogic.csproj

```diff
@@ -5,6 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="AsyncAwaitBestPractices" Version="5.0.2" />
```

Comment:
what is provided by this nuget package?

Reply:
A method that is providing a SafeFireAndForget-Method. The other team defined for them to use it.